app/views/games/show.html.erb

```diff
@@ -1 +1,22 @@
-<h1>Game's show page</h1>
+<div class="row">
+  <div class="medium-4 medium-offset-4 columns">
+  	<h1>Have Fun</h1>
+			<table class="chessboard" >
+				<% (0..7).each do |row| %>
+					<tr>
+					<% (0..7).each do |col|  %>
+						<% if ((row % 2 == 0) && (col % 2 != 0)) || ((row % 2 != 0) && (col % 2 == 0)) %> 
```

Comment:
We like to see very little 'business logic' in the views.  Could you create a helper for this?